Source/Core/DolphinNoGUI/DolphinNoGUI.vcxproj

```diff
@@ -18,8 +18,9 @@
   <ItemDefinitionGroup>
     <Link>
       <AdditionalDependencies>avrt.lib;iphlpapi.lib;winmm.lib;setupapi.lib;rpcrt4.lib;comctl32.lib;Shlwapi.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalDependencies Condition="'$(Platform)'=='x64'">opengl32.lib;avcodec.lib;avformat.lib;avutil.lib;swresample.lib;swscale.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories Condition="'$(Platform)'=='x64'">$(ExternalsDir)ffmpeg\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalDependencies Condition="'$(Platform)'=='x64'">opengl32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>libavcodec.a;libavformat.a;libavutil.a;libswresample.a;libswscale.a;Bcrypt.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
`.a`? Seems like that's correct according to the submodule, but that's rather unusual for Windows.

Reply:
the `configure` script of ffmpeg chooses this filename format when built under wsl. The file content is the same. We could rename it but I'm not aware it's a problem.

Reply:
Fair enough.

Reply:
fwiw, it turns out to be a problem with dolphin's existing `CMake/FindFFmpeg.cmake`, and i'd rather just rename the files to be more windows-y than fiddle with the cmake